sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -135,6 +135,12 @@ public int MaxConcurrentCalls
         /// </value>
         public SubQueue SubQueue { get; set; } = SubQueue.None;
 
+        /// <summary>
+        /// Gets the ID to identify this client. This can be used to correlate logs and exceptions.
+        /// </summary>
+        /// <remarks>Every new client has a unique ID.</remarks>
```

Comment:
Do we actually enforce this? What if a user uses the same identifier for multiple processors? Would that cause any issues?

Reply:
We don't _(and we probably  probably should be careful about stating it this way)_ - and it should not cause problems outside of log readability or durable terminus state _(eventually)_.